docs/pages/database-access/guides/gui-clients.mdx

```diff
@@ -90,57 +180,69 @@ for password, leave the password field empty and click OK.
 
 ## MySQL Workbench
 
-[MySQL Workbench](https://www.mysql.com/products/workbench/) is a visual tool
-that provides comprehensive MySQL administration and SQL development tools.
+[MySQL Workbench](https://www.mysql.com/products/workbench/) is a GUI
+application that provides comprehensive MySQL administration and SQL development
+tools.
 
-In MySQL Workbench "Setup New Connection" dialog, fill out "Connection Name",
-"Hostname", "Port", and "Username":
+In the MySQL Workbench "Setup New Connection" dialog, fill out "Connection
+Name", "Hostname", "Port", and "Username":
 
 ![MySQL Workbench
 Parameters](../../../img/database-access/workbench-parameters@2x.png)
 
-In the "SSL" tab, set "Use SSL" to `Require and Verify Identity` and enter paths
-to  your CA, certificate, and private key files from `tsh db config` command:
+In the "SSL" tab, set "Use SSL" to `Require and Verify Identity` and enter the
+paths to your CA, certificate, and private key files (see
+[Get connection information](./gui-clients.mdx#get-connection-information)):
 
 ![MySQL Workbench SSL](../../../img/database-access/workbench-ssl@2x.png)
 
 Optionally, click "Test Connection" to verify connectivity:
 
 ![MySQL Workbench Test](../../../img/database-access/workbench-test@2x.png)
 
-Save the connection, and connect to the database.
+Save the connection and connect to the database.
 
 ## MySQL DBeaver
 
 <Admonition type="note">
-  Teleport's DBeaver MySQL integration only supports MySQL server (=mysql.dbeaver_min_ver=) or older.
+
+  Teleport's DBeaver MySQL integration only supports MySQL server
+  (=mysql.dbeaver_min_ver=) or older.
```

Comment:
Should we say "only supports versions up to (=mysql.dbeaver_min_ver=)" instead?

As I was reading this sentence, I was almost certain it was going to end with "or newer" just because that's normally how compatibility notes read. This one caught me by surprise.